main.go

```diff
@@ -633,7 +639,7 @@ func main() {
 			{
 				Name:      "favorite",
 				Aliases:   []string{"fav", "f"},
-				Usage:     "Quickly access a favorite",
+				Usage:     "Quickly access a favorite via federating into the web console or from the cli via a stak.",
```

Comment:
This is being nit picky but if we can shorten this up it will better fit in constrained editors. Usual target is around 80 chars wide. It already has ~20 chars proceeding it.